### ISSUE TITLE
Multi-account

### ISSUE THREAD

**Issue by nadonomy** [vector-im/roadmap#11]
__Summary__
Today, it's not possible to sign in to multiple Matrix accounts in Element, which isn't ideal for users who want to do this, especially on mobile, and furthermore those participating in lots of encrypted rooms.

__Intended outcome__
Implement the ability to sign in to multiple accounts and switch between them on all platforms.

**Comment by BloodyIron** [comment]
Can we get this assigned to a human? This has been a requested feature since 2017, and has been in the "Next" section since Nov 2020.

**Comment by seconddayout** [comment]
Why has this been demoted from `Next` to `Later`? What needs to happen to get the Element team to prioritize this?

**Comment by github-tomster** [comment]
@seconddayout 
consider that 'multi account' feature is something so old like the 'messenger-era' itself.
to state something like: `An open network for secure, decentralized communication`
and at the same time preventing multi-account 🤦🏻‍♂️ 
**failure by design one could think** 🤷🏻‍♂️ 
this is all a joke 🤡 

btw: anyone here with more than one emailaddress 🤣 

tl;dr: 'developing' something like this and not even considering multi-account **from scratch**... wtf

**Comment by coffeemesh** [comment]
Since almost everybody is opening their own servers, it keeps requiring me to sign in to different accounts all the time. This feature would be godsent!

**Comment by xRealNeon** [comment]
> Since almost everybody is opening their own servers, it keeps requiring me to sign in to different accounts all the time. This feature would be godsent!

You know that you don't need an account for every server?
You can send a message from one server to another like an email.

@coffeemesh

**Comment by almereyda** [comment]
There are servers which decidedly do not federate.

**Comment by BloodyIron** [comment]
> > Since almost everybody is opening their own servers, it keeps requiring me to sign in to different accounts all the time. This feature would be godsent!
> 
> You know that you don't need an account for every server? You can send a message from one server to another like an email.
> 
> @coffeemesh

That doesn't mean there isn't valid functional need here. There is.

**Comment by trymeouteh** [comment]
Cannot wait for this on desktop and mobile.

**Comment by nik2323** [comment]
Очень нужная вещь чтобы не пользоваться 2 разных приложения.

**Comment by urielarg** [comment]
Why no one at Matrix seems to care about this issue??? at least write something for the community who actually cares to report this feature request...

**Comment by ptman** [comment]
@urielarg There are clients other than element that have multi-account support https://matrix.org/clients-matrix/

**Comment by Jan02** [comment]
multi-account support will bring more users.

**Comment by genofire** [comment]
Dissussion:
 - global: https://github.com/vector-im/element-meta/discussions/1681
 - Element X: https://github.com/vector-im/element-meta/discussions/1832

Issues Collection per Application (i do not list all the duplicates):
 - Android:
   - X: https://github.com/vector-im/element-x-android/issues/190
   - current: https://github.com/vector-im/element-android/issues/26
   - Riot: https://github.com/vector-im/riot-android/issues/120
 - iOS:
   - X: https://github.com/vector-im/element-meta/issues/1831
   - current: https://github.com/vector-im/element-ios/issues/315
 - Web: https://github.com/vector-im/element-web/issues/2320
 
 Old Meta:
 - https://github.com/vector-im/riot-meta/issues/122

**Comment by DraconicNEO** [comment]
Seriously this is a critical feature for a decentralized platform like Matrix where it is almost guaranteed that users will Have multiple accounts. Not being able to sign into multiple accounts is a huge detriment, especially since you can't exactly log out of your accounts to switch between them since you need to keep the account logged in in at least one place else you lose your E2E keys and then can't retrieve older encrypted messages.

**Comment by vollkorntomate** [comment]
Shouldn’t such a feature be considered from the very beginning, as it determines quite a lot of the underlying architecture? Having multiple accounts means the app needs to manage multiple encryption keys separately, but still somehow handle decryption in order to show notifications for each account.
Changing these very basic things later on seems like quite an effort and I’m worried that this issue will still be open for another six years.

**Comment by Coehill** [comment]
Since it's been nearly a year, is there any update on this?

**Comment by BloodyIron** [comment]
If you read this board: https://github.com/element-hq/roadmap/projects/1

You'll see they "stopped using this in 2022"... yet didn't tell anyone beyond an area people don't seem to even be aware of (not that I really was myself).

**Comment by jittygitty** [comment]
@BloodyIron So where is the "new" location for the Element "Roadmap" and progress etc?

Also, any "estimate" on when this feature may land in any Element client, whether mobile or Desktop?

**Comment by BloodyIron** [comment]
> @BloodyIron So where is the "new" location for the Element "Roadmap" and progress etc?
> 
> Also, any "estimate" on when this feature may land in any Element client, whether mobile or Desktop?

Why are you asking me? I'm not a Dev or contributor to the code to date.

**Comment by jittygitty** [comment]
Sorry somehow I figured you might know where the new Roadmap was, i thought i was the only lost one, not knowing.

The other question about when this feature might land, was not to you specifically but to whoever would know. Apologies for the confusion, maybe should of been two different posts. (been sick so brain's likely not all there yet!)

**Comment by BloodyIron** [comment]
Oh well sorry I don't think I know where any reliable Roadmap could be, or is. At this point the devs are so quiet it's probably very hard to tell :/ (I'd love to be wrong though!)

Hope you get better! ❤️
